action.yml

```diff
@@ -83,9 +90,20 @@ runs:
           runs/wokwi/reports/metrics.csv
           runs/wokwi/reports/synthesis/1-synthesis*
 
-    - name: Create PNG
+    # Create and store PNG...
+    - name: Render SVG from GDS
+      shell: bash
+      run: ./tt/tt_tool.py --create-svg
+
+    - name: Convert SVG to PNG
       shell: bash
-      run: ./tt/tt_tool.py --create-png
+      run: rsvg-convert --unlimited gds_render.svg -s <(echo 'text{display:none;}') -o gds_render_png24.png --no-keep-image-data
+
+    - name: Shrink PNG
+      shell: bash
+      run: |
+        pngquant --quality 10-30 --speed 1 --nofs --strip --output gds_render.png gds_render_png24.png
+        ls -al gds_render*.png
```

Comment:
Any reason to keep this `ls -al` here?

Reply:
Probably don't need to keep it. It was in there as a way to show the size difference between original PNG and quantised (recompressed) PNG, mostly for diagnostics.

Reply:
Alright, can you please remove it then?

Reply:
OK, removed in https://github.com/TinyTapeout/tt-gds-action/pull/1/commits/bae56eec67d05458b812492680366a35f5617cf3